wsidicom/wsidicom.py

```diff
@@ -2084,3 +2084,34 @@ def _validate_collection(
             if item.uids is not None and item.uids != slide_uids:
                 raise WsiDicomMatchError(str(item), str(self))
         return slide_uids
+
+    @classmethod
```

Comment:
This is a staticmethod and not a classmethod.

Reply:
No, it uses cls._get_filepaths() and cls._filter_paths().